### PR TITLE
Simplify activation of WheelInput and MoveInput.

### PR DESCRIPTION
**Pull request by ericsoco**
Add missing devDep.
Note that tests are failing, apparently due to a transpilation error, so this PR probably won't pass CI.

### CODE REVIEW

File: src/utils/events/move-input.js [hunk 1]
```diff
@@ -25,12 +25,14 @@ export default class MoveInput {
...
-    Object.assign(this.options, options);
-  }
-
-  isSourceOf(eventName) {
-    return MOVE_EVENT_TYPES.indexOf(eventName) >= 0;
+  /**
+   * Enable this input (begin processing events)
+   * if the specified event type is among those handled by this input.
+   */
+  enableOnEventType(eventType) {
```
Comment (ibgreen) [inline src/utils/events/move-input.js]: Should we drop the `On` and just call this `enableEventType`? Are we saying something important with the `On`?

Do we only need the ability to enable event types (not disable them)?

File: package.json [hunk 2]
```diff
@@ -85,6 +85,7 @@
     "react-addons-test-utils": "^15.4.2",
     "react-dom": "^15.4.0",
     "reify": "^0.4.4",
+    "source-map-loader": "^0.2.1",
```
Comment (ibgreen) [inline package.json]: Didn't @shaojingli already add this? Maybe rebase?

Comment (ericsoco) [inline package.json]: It was added in a followup this morning, wasn't there last night.

File: src/utils/events/move-input.js (hunk 1 above)
Comment (ericsoco) [inline src/utils/events/move-input.js]: I can't find a good name for this function. We're not enabling for a specific event type, we're enabling the entire input if the passed event type is one that the input handles.

As for disabling, yes, I suppose we might need to (though I'd say it's fairly unlikely); I added @Pessimistress' `set()` method back in to support that.

**Pull request merged**